chat/fe/src/views/RoomView.ts

```diff
@@ -234,6 +235,7 @@ export function RoomView(props: RoomViewProps) {
         setStyle(option, {
           margin: '0',
           padding: '8px 12px',
+          overflowWrap: 'Normal',
```

Comment:
What does this do?  Is this needed to effectively undo the overflowWrap from above?  If so, it might be better to target the overflowWrap: "anywhere" at the element specifically with the text